### ISSUE TITLE
Metaquast provide multiple gff files

### ISSUE THREAD

**Issue by crarlus** [ablab/quast#40]
Hi,
I was wondering if it was possible to supply multiple gff files when calling metaquast? There is an option for supplying multiple fasta files (with the "-R" flag). 
So what about "-G" with one gff corresponding to each genome provided in "-R"?

**Comment by alexeigurevich** [comment]
Hi,
Currently it is not available, -G accepts a single file only. However, this feature request looks useful, so we will try to add this to metaQuast soon.
Currently you can merge all your GFFs into a single file and provide it with -G. Note that metaQuast will assume that the reference sequence names in coordinates are started with prefix equal to FASTA filename. E.g. if you run 
`./metaquast.py -R test_data/reference1.fasta.gz -R test_data/reference2.fasta.gz  assembly.fa -G genes.gff`
than the entries in GFF should start with "reference1_" and "reference2_". E.g.
```
reference1_Ecoli   .  gene    5683    6459    .       -       .       ID=9
reference2_Lactobacillus_plantarum     .       gene    5683    6459    .       -       .       ID=9
```

**Comment by alexeigurevich** [comment]
The support for multiple genomic features is added to the master branch and will be included in the upcoming v5.0 release.
Note that we also improved support of feature types in GFF. We added `--features` option instead of `-G` (it is still available but deprecated now). Now you can either ask QUAST to count ALL features from GFF file (simple `--features /path/to/your.gff`) or ask to count a specific genomic feature, e.g. 'gene' (prepend feature name to the file path and separate them by a colon: `--features gene:/path/to/your.gff`).
